test/cppsuite/src/main/test.cpp

```diff
@@ -168,11 +168,8 @@ test::run()
         it->finish();
 
     /* Validation stage. */
-    if (_operation_tracker->enabled()) {
-        std::unique_ptr<configuration> tracking_config(_config->get_subconfig(OPERATION_TRACKER));
```

Comment:
Sidenote, no action required: It seems like we don't have compiler warnings for unused variables, and looking in the cmake/ folder we don't configure any warnings in `COMPILER_DIAGNOSTIC_CXX_FLAGS`.  [WT-8247](https://jira.mongodb.org/browse/WT-8247) is in the defined pipeline to add warnings for cpp code.

Reply:
fb5c7b9ebe7efe43860c87692292d32d3ec9bd50